style.css

```diff
@@ -77,6 +77,7 @@ form label {
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
+  margin-bottom: 100px;
```

Comment:
I recommend changing .container's padding-bottom value to 100px instead of adding a margin-bottom in the .cat-pictures